INSTALL.md

```diff
@@ -0,0 +1,315 @@
+# Installation
+
+## Requirements
+* [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or higher.
+* [Java Cryptography Extension](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html) to
+  use Client-side Encryption with stronger ciphers (192/256-bit).
+* [Maven 3.3.x](https://maven.apache.org/) to contribute to the project.
```

Comment:
I think we can just say "Maven" at this point.  Also `3.3.x` doesn't match `maven.min.version`

Reply:
This should be Maven 3.1